test/test_lower.c

```diff
@@ -2008,7 +2008,9 @@ START_TEST(test_cover)
 
    tree_t e = run_elab();
 
-   cover_tagging_t *tagging = cover_tags_init(COVER_MASK_ALL);
+   cover_opts_t cov_opts;
+   cov_opts.mask = COVER_MASK_ALL_TYPES;
```

Comment:
This leaves `cov_opts.array_limit` uninitialised. Use a C99-style designated initialiser instead which zeros out the other members:

```c
cover_opts_t cov_opts = { .mask = COVER_MASK_ALL_TYPES };
```

But I'd rather we just pass two separate arguments.

Reply:
Done.